scanpipe/tests/test_forms.py

```diff
@@ -57,3 +58,22 @@ def test_scanpipe_forms_inputs_base_form_input_urls(self, mock_get):
         self.assertEqual(["archive.zip"], self.project1.input_files)
         expected = {"archive.zip": "https://example.com/archive.zip"}
         self.assertEqual(expected, self.project1.input_sources)
+
+    @mock.patch("requests.get")
+    def test_scanpipe_forms_project_form_name(self, mock_get):
+        data = {
+            "name": "  Test   Name   ",
+            "input_urls": "https://example.com/archipe.zip",
+            "pipeline": "scan_codebase",
+        }
+
+        mock_get.side_effect = None
+        mock_get.return_value = mock.Mock(
+            content=b"\x00",
+            headers={},
+            status_code=200,
+            url="url/archive.zip",
+        )
+        form = ProjectForm(data=data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual("Test Name", form.cleaned_data["name"])
```

Comment:
It would be nice to save() the form and load the new project from the database to ensure the "saved "name was properly cleaned as well.